dbt_copilot_helper/commands/application.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/env python
+
+import time
+from datetime import datetime
+from datetime import timedelta
+
+import click
+
+from dbt_copilot_helper.utils.aws import get_aws_session_or_abort
+from dbt_copilot_helper.utils.click import ClickDocOptGroup
+from dbt_copilot_helper.utils.versioning import (
+    check_copilot_helper_version_needs_update,
+)
+
+
+def get_query_results(env, app, profile, query_string):
+    project_session = get_aws_session_or_abort(profile)
+
+    click.secho(
+        f"Showing status for app {app} in aws account {profile}",
+        fg="green",
+    )
+    logs_client = project_session.client("logs")
+    ecs_client = project_session.client("ecs")
+
+    response = ecs_client.list_clusters()
+
+    # Find clusters log-group
+    for cluster in response["clusterArns"]:
+        cluster_name = cluster.split("/")[-1]
+        APP = cluster_name.split("-")[0]
+        ENV = cluster_name.split("-")[1]
+        if app == APP and env == ENV:
+            log_group_name = f"/aws/ecs/containerinsights/{cluster_name}/performance"
+            break
+
+    # Container stats are 5 mins behind realtime.
+    date_time = datetime.now()
+    end_time = int(date_time.timestamp())
+    start_time = int((date_time - timedelta(minutes=5)).timestamp())
+    end_time = int((date_time - timedelta(minutes=4)).timestamp())
+
+    click.echo(
+        click.style(
+            "Date & Time:  ",
+            fg="cyan",
+        )
+        + click.style(f"{datetime.utcfromtimestamp(start_time)}", fg="cyan", bold=True)
+    )
+
+    # First you need to create a log query, then retrieve the log query.
+    cpu_response_id = logs_client.start_query(
+        logGroupName=log_group_name,
+        startTime=start_time,
+        endTime=end_time,
+        queryString=query_string,
+    )
+
+    # Need to wait approx 5 seconds for the query to be available.
+    click.secho(
+        "waiting 5s...",
+        fg="cyan",
+    )
+    time.sleep(5)
+
+    cpu_response = logs_client.get_query_results(queryId=cpu_response_id["queryId"])
```

Comment:
So it's faster where it can be and less brittle if it needs longer than 5 seconds, could we do this on a loop with a 1 second sleep and maybe a 15 second timeout.

Reply:
good suggestion, will make that change

Reply:
Added check and wait